samples: matter: Created configuration with LEDs disabled (#18080)
nrfconnect/sdk-nrf: doc/nrf/protocols/matter/getting_started/low_power_configuration.rst

   :start-after: disable_unused_pins_start
   :end-before: disable_unused_pins_end

Disable LEDs module
===================

> When performing the power measurements on variouis development kits, the LEDs can be included in the measurement circuit or not:

Reviewer: ```suggestion
When performing power measurements on various development kits, the LEDs can either be included in the measurement circuit or not:
```

> * For the nRF52840 DK and nRF5340 DK the LEDs are excluded from the measurement circuit, so they can be enabled for the low power configuration and it is not going to impact the measurement results.

Reviewer: ```suggestion
* For the nRF52840 DK and nRF5340 DK, the LEDs are excluded from the measurement circuit, so they can be enabled for low power configuration without impacting the measurement results.
```

> * For the nRF54L15 DK, the MOSFET transistors controlling the LEDs are included in the measurement circuit, what results in increased measurement result by an additional, small leakage current, appearing if the LED is turned on.

Reviewer: ```suggestion
* For the nRF54L15 DK, the MOSFET transistors controlling the LEDs are included in the measurement circuit, which results in increased measurement results due to a small leakage current if the LED is turned on.
```

> * For the nRF54L15 DK, the MOSFET transistors controlling the LEDs are included in the measurement circuit, what results in increased measurement result by an additional, small leakage current, appearing if the LED is turned on.

Reviewer 2: ```suggestion
* For the nRF54L15 DK, the MOSFET transistors controlling the LEDs are included in the measurement circuit.
  This results in measurement results being increased by an additional, small leakage current that appears if an LED is turned on.
```

>   To measure the current consumption of the nRF54L15 SoC and not include development kit components, such as LEDs, it is recommended to disable them.

Reviewer: ```suggestion
  To measure the current consumption of the nRF54L15 SoC without including development kit components, such as LEDs, it is recommended to disable them.
```

> To disable LEDs from the Matter samples and applications, set the :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_LEDS` Kconfig option to ``n``.

Reviewer: ```suggestion
To disable LEDs in the Matter samples and applications, set the :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_LEDS` Kconfig option to ``n``.
```
